gandalf.js

```diff
@@ -1,31 +1,32 @@
-const isEven = number % 2 === 0;
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.